easybuild/easyconfigs/z/zsh/zsh-5.8-foss-2019b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'zsh'
+version = '5.8'
+
+homepage = 'http://www.zsh.org/'
+description = "Zsh is a shell designed for interactive use, although it is also a powerful scripting language."
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
```

Comment:
@levlafayette Since you're using `GCC/8.3.0` as toolchain, the easyconfig file should be renamed to `zsh-5.8-GCC-8.3.0.eb`.